src/processTargets/modifiers/scopeTypeStages/ParagraphStage.ts

```diff
@@ -21,10 +21,9 @@ export default class implements ModifierStage {
 
   private getEveryTarget(target: Target): ParagraphTarget[] {
     const { contentRange, editor } = target;
-    const { isEmpty } = contentRange;
     const { lineCount } = editor.document;
-    const startLine = isEmpty ? 0 : contentRange.start.line;
-    const endLine = isEmpty ? lineCount - 1 : contentRange.end.line;
+    const startLine = target.hasExplicitRange ? contentRange.start.line : 0;
+    const endLine = target.hasExplicitRange ? contentRange.end.line : lineCount;
```

Comment:
Why was there a `- 1` here?  It caused a bug when the selection ended within the first line of a block

Reply:
Oh ok wow this is all messed up.  One sec fixing